ci/build_test_OnCommit.groovy

```diff
@@ -21,7 +21,7 @@ pipeline {
               } else {  // branches without PR - check changes in last commit
                 git_diff = sh (script: "git diff --name-only HEAD^..HEAD", returnStdout: true).trim()
               }
-              def matched = (git_diff =~ /src|third_party|(\n|^)Dockerfile|(\n|^)Makefile|\.c|\.h|BUILD|WORKSPACE|(\n|^)rununittest\.sh/)
```

Comment:
Why are we removing "BUILD"? There are files called "BUILD" that should require image rebuild.
- BUILD
- BUILD.bazel